dynamicprefixvici.c

```diff
@@ -157,19 +234,180 @@ static void ParseCommandLineArguments(int argc, char** argv, CommandLineArgument
         }
     }
 
-    // Check whether a pool name and a prefix is defined
-    if(arguments->pool_name == NULL && arguments->prefix == NULL)
+    // Check whether a pool name and a prefix_address is defined
+    if(   arguments->pool_name == NULL 
```

Comment:
("whitespace here" arguments->pool_name

remove whitespace

Reply:
The whitespace increases readability